### PR TITLE
Add test for update affiliate stats trigger

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/719?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#719** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/719?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#717** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/717?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: supabase/migrations/20240919082056_create_affiliate_stats_table.sql [hunk 1]
```diff
@@ -1,5 +1,5 @@
 CREATE TABLE "public"."affiliate_stats"(
-    "paymaster_tx_count" integer NOT NULL DEFAULT 0 ::integer,
+    "paymaster_tx_count" integer NOT NULL DEFAULT 0 ::int8,
```
Comment (0xBigBoss) [inline supabase/migrations/20240919082056_create_affiliate_stats_table.sql:2]: Why this

Comment (youngkidwarrior) [reply supabase/migrations/20240919082056_create_affiliate_stats_table.sql:2]: mistake

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Sep 22, 4:03 AM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/719).

**Pull request merged**